### PR TITLE
Add FaceID permission setting

### LINKED ISSUE #58: FaceID needs permissions.
**Description**
AppleSimUtils Touch/FaceID support is missing the ability to toggle the OS level permissions, in the same way that camera, location, etc need OS level permissions.

**Steps to Reproduce**
1. Add FaceID to your app (in our case, we're using `react-native-touch-id`) - code here:

```
LAContext *context = [[LAContext alloc] init];
    NSError *error;
    
    // Check to see if we have a passcode fallback
    NSNumber *passcodeFallback = [NSNumber numberWithBool:true];
    if (RCTNilIfNull([options objectForKey:@"passcodeFallback"]) != nil) {
        passcodeFallback = [RCTConvert NSNumber:options[@"passcodeFallback"]];
    }
    
    if ([context canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics error:&error]) {
        
        // No error found, proceed
        callback(@[[NSNull null], [self getBiometryType:context]]);
    } else if ([passcodeFallback boolValue] && [context canEvaluatePolicy:LAPolicyDeviceOwnerAuthentication error:&error]) {
        
        // No error
        callback(@[[NSNull null], [self getBiometryType:context]]);
    }
    // Device does not support FaceID / TouchID / Pin OR there was an error!
    else {
        if (error) {
            NSString *errorReason = [self getErrorReason:error];
            NSLog(@"Authentication failed: %@", errorReason);
            
            callback(@[RCTMakeError(errorReason, nil, nil), [self getBiometryType:context]]);
            return;
        }
        
        callback(@[RCTMakeError(@"RCTTouchIDNotSupported", nil, nil)]);
        return;
    }
```
2. Attempt to use FaceID
3. You're met with a prompt like the following:
Image from iOS

**Expected Behavior**
It would be ideal with we could set the permissions at the command line level, in the same way we set permissions for camera, location, etc.

### PR DESCRIPTION
**Pull request by brandons**
Adds ability to toggle FaceID permission. @LeoNatan Can I please get a review? Thank you!

Closes #58

### CODE REVIEW

**Comment by LeoNatan** [comment]
Thanks, will review shortly. I assume you tested this and the prompt in the issue does not appear after setting this permission?

File: README.md [hunk 1]
```diff
@@ -62,4 +62,5 @@ Available Permissions:
     reminders=YES|NO|unset
     siri=YES|NO|unset
     speech=YES|NO|unset
+    faceId=YES|NO|unset
```
Comment (LeoNatan) [inline README.md]: Pls name this "faceid", in line with other permission names

File: applesimutils/applesimutils/main.m [hunk 2]
```diff
@@ -233,6 +233,7 @@ static BOOL performPermissionsPass(NSString* permissionsArgument, NSString* simu
 																   @"motion": @"kTCCServiceMotion",
 																   @"siri": @"kTCCServiceSiri",
 																   @"speech": @"kTCCServiceSpeechRecognition",
+                                                                   @"faceId": @"kTCCServiceFaceID",
```
Comment (LeoNatan) [inline applesimutils/applesimutils/main.m]: Please indent with tabs, rather than spaces. Thanks

File: applesimutils/applesimutils/main.m [hunk 3]
```diff
@@ -450,6 +451,7 @@ int main(int argc, const char* argv[]) {
 												 @"reminders=YES|NO|unset",
 												 @"siri=YES|NO|unset",
                                                  @"speech=YES|NO|unset",
+                                                 @"faceId=YES|NO|unset",
```
Comment (LeoNatan) [inline applesimutils/applesimutils/main.m]: Ditto. Please indent with tabs, rather than spaces. Thanks

**Comment by brandons** [comment]
@LeoNatan Thanks! Changes made. And yes, it is correctly bypassing the Face ID permission prompt for me.

**Comment by LeoNatan** [comment]
Thanks!

**Pull request merged**